AsyncDisplayKit/Details/Transactions/_ASAsyncTransaction.h

```diff
@@ -95,6 +97,25 @@ typedef NS_ENUM(NSUInteger, ASAsyncTransactionState) {
                    completion:(asyncdisplaykit_async_transaction_operation_completion_block_t)completion;
 
 /**
+ @summary Adds a synchronous operation to the transaction.  The execution block will be executed immediately.
+ 
+ @desc The block will be executed on the specified queue and is expected to complete synchronously.  The async
+ transaction will wait for all operations to execute on their appropriate queues, so the blocks may still be executing
+ async if they are running on a concurrent queue, even though the work for this block is synchronous.
```

Comment:
@appleguy, I just copied the entire comment from original method when adding variant with priority. There are two ways to schedule transaction operation, but the async one (which requires scheduled block to call completion block) is not used anywhere as far as I can tell.